addons/account/test/account_minimal_test.xml

```diff
@@ -29,7 +34,7 @@
             <field name="code">X1011</field>
             <field name="name">Purchased Stocks - (test)</field>
             <field name="user_type_id" ref="account.data_account_type_current_assets"/>
-            <field name="tag_ids" eval="[(6,0,[ref('account.demo_stock_account')])]"/>
+            <field name="tag_ids" eval="[(6,0,[ref('demo_stock_account')])]"/>
```

Comment:
why do you need to patch account?

Reply:
here in stock_landed_costs https://github.com/odoo/odoo/pull/34976/commits/cf348ce126a3b7a27ab8d3e745988c7b64144bf7#diff-d1c56263f47d7df30bef4ef05fd6ee4cR16 and stock_dropshipping https://github.com/odoo/odoo/pull/34976/commits/910d389d2f5afac55c514faf87048ffd9572ee89#diff-3158699eaf3caca44e2531217badae2eR19
we are loading account_minimal_test file, in that we have used 'demo_stock_account' as a reference, but it is in account_demo.xml file https://github.com/odoo/odoo/blob/master/addons/account/demo/account_demo.xml#L10 and goal of this task is to pass test cases without demo data, so it is giving us reference error, and that reference is not used anywhere else, so we removed from demo.xml and moved it to test file.